DOCS-3207: Update refunds FAQ article (#2077)
MultiSafepay/docs: content/integration/ready-made/prestashop-1-7/faq/processing-refunds.md

    - /integrations/prestashop-1-7/faq/refunding-prestashop/
    - /payments/integrations/ecommerce-platforms/prestashop-1-7/faq/processing-refunds/
---
**Refunds**

- From your [MultiSafepay dashboard](/refunds/full-partial/): Full and partial refunds for all payment methods 
- From your PrestaShop 1.7 [backend](/glossaries/multisafepay-glossary/#backend):  
    - Full and partial refunds
    - [Pay later refunds](/payment-methods/pay-later/) **not** supported
    

> When doing a refund through the PrestaShop [backend](/glossaries/multisafepay-glossary/#backend) it will automatically process this refund through the MultiSafepay API. If you choose to use a voucher refund, it will not process this refund through the MultiSafepay API.

Reviewer: ```suggestion
Refunds initiated in your PrestaShop [backend](/glossaries/multisafepay-glossary/#backend) are automatically processed via our API, **except** for voucher refunds.
```
